src/components/Pagination/Pager.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
```

Comment:
ugh one more thing now... we are rebasing everything to `classNames` soon in #274 ... do you mind updating this one last thing? I am so sorry 😆